common/persistence/persistence-tests/historyV2PersistenceTest.go

```diff
@@ -174,6 +181,82 @@ func (s *HistoryV2PersistenceSuite) TestScanAllTrees() {
 	s.Equal(0, len(trees))
 }
 
+func (s *HistoryV2PersistenceSuite) TestHistoryScavenger() {
```

Comment:
I think we are deprecating tests under `persistence/persistence-test` folder. New tests should be added to `persistence/tests`.